stubs/boltons/boltons/ecoutils.pyi

```diff
@@ -0,0 +1,34 @@
+# TODO: DONE!
+
+from typing import Any, NoReturn
+
+ECO_VERSION: str
+PY_GT_2: bool
+getrandbits: Any
+HAVE_URANDOM: bool
+INSTANCE_ID: str
+IS_64BIT: bool
+HAVE_UCS4: bool
+HAVE_READLINE: bool
+SQLITE_VERSION: str
+OPENSSL_VERSION: str
+TKINTER_VERSION: str
+ZLIB_VERSION: str
+EXPAT_VERSION: str
+CPU_COUNT: int
+HAVE_THREADING: bool
+HAVE_IPV6: bool
+RLIMIT_NOFILE: int
+RLIMIT_FDS_SOFT: int
+RLIMIT_FDS_HARD: int
+START_TIME_INFO: dict[str, str | float]
+
+def get_python_info() -> dict[str, Any]: ...
+def get_profile(**kwargs) -> dict[str, Any]: ...
+def get_profile_json(indent: bool = ...) -> str: ...
+def main() -> NoReturn: ...
+
+# def dumps(val: Any, indent: int) -> str: ...
```

Comment:
Looks like this does exist: https://github.com/mahmoud/boltons/blob/6c7fa3e05659fac6515f661153c89d4ad02df168/boltons/ecoutils.py#L360-L363

Reply:
Yes, there exists.
But `stubtest` complains about it in runtime https://github.com/python/typeshed/actions/runs/3523498827/jobs/5907729287#step:5:50

Reply:
In this case it is better to add it to `@tests/stubtest_allowlist.txt` with a comment.

Reply:
DONE!